### PR TITLE
fix: issue on RTL support checkbox component 

### LINKED ISSUE #2465: [BUG] - Lack of RTL Support in checkbox
### NextUI Version

2.0.10

### Describe the bug

The repository does not work correctly when the direction is set to RTL (Right-to-Left). Elements are not displayed as expected, leading to usability issues.

### Your Example Website or App

_No response_

### Steps to Reproduce the Bug or Issue

Go to [this page](https://storybook.nextui.org/?path=/story/components-checkbox--default&globals=locale:ar-AE)
Change the document direction to RTL (locale: Arabic).
Observe the issues mentioned above.

### Expected behavior

I expected the UI to properly support RTL direction, with elements aligning and displaying correctly from right to left.

### Screenshots or Videos

_No response_

### Operating System Version

All

### Browser

Chrome

### PR DESCRIPTION
**Pull request by mrbadri**
Closes #2465

## 📝 Description

Add RTL support to the checkbox component.

## ⛳️ Current behavior (updates)

The checkbox component currently lacks support for right-to-left (RTL) direction.

## 🚀 New behavior

This PR addresses the RTL support issue in the checkbox component, ensuring correct rendering in RTL environments.

## 📝 Additional Information
close #2465

### CODE REVIEW

File: .changeset/clean-tools-double.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@nextui-org/checkbox": major
```
Comment (jrgarciadev) [inline .changeset/clean-tools-double.md:2]:
```suggestion
"@nextui-org/checkbox": patch
```

**Review by jrgarciadev — CHANGES_REQUESTED** [review]
Thank you @mrbadri  🙏🏻 , please adjust the changeset

File: .changeset/clean-tools-double.md (hunk 1 above)
Comment (mrbadri) [reply .changeset/clean-tools-double.md]: Hi [jrgarciadev](https://github.com/jrgarciadev) ,
I resolved this
[jrgarciadev](https://github.com/jrgarciadev)

**Comment by mrbadri** [comment]
> Thank you @mrbadri 🙏🏻 , please adjust the changeset

Hi  [jrgarciadev](https://github.com/jrgarciadev), 
i resolves this 
https://github.com/nextui-org/nextui/pull/2466/commits/d2cd0b8ce4803c44d06dd429b2df9fef10cb02b8

**Review by jrgarciadev — APPROVED** [review]

**Review by wingkwong — COMMENTED** [review]
please put the issue number next to `Closes #` (i.e. `Closes #2465`) instead of mentioning in additional info so that the issue can be closed automatically once it is merged.

**Comment by mrbadri** [comment]
> wingkwong

Hi [wingkwong](https://github.com/wingkwong), 
i resolves this

**Review by wingkwong — APPROVED** [review]

**Comment by mrbadri** [comment]
Hi [jrgarciadev](https://github.com/jrgarciadev),

I hope you're doing well! I've made some updates to address the feedback you provided earlier. Could you please take another look at my pull request when you have a moment? Your feedback is greatly appreciated.

Thank you!

Mohammad Reza Badri

**Pull request merged**